src/components/DataTable/index.js

```diff
@@ -52,10 +63,20 @@ export function DataTable ({
 		}, 500);
 	}, [filterValue]);
 
+	// rounding page number divisible by 5
+	function round(x) {
+		const rowCount = Math.round(x / 5) * 5;
```

Comment:
@Aringoaway To make it more defensive, you need to ensure that value `x` is an integer (not string, not boolean, not undefined, etc..). If not, return default limit